docs/datasets/index.rst

```diff
@@ -52,10 +52,83 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
+weighing by the exposure.
+
+For the model evaluation, an important factor that needs to be accounted for is
+that the energy threshold changes between obseravtions.
+To ensure that the npred (ie, the predicted number of counts) on the stacked
+dataset is the sum expected by stacking the npred of the individual runs,
+a `~gammapy.irf.EDispersionMap` is used, which contains the
+The mask_safe from each dataset is applied on the respective reconstructed energy axis
+of the energy dispersion matrix, and the masked matrices are combined.
+Values lying outside the safe mask of each dataset are lost.
+
+Stacking of multiple datasets is implemented as follows.
+Here, :math:`k` denotes a bin in reconstructed energy,
+:math:`l` a bin in true energy and
+:math:`j` is the dataset number
+
+
+================= ================================== ==================================================================================
+Dataset attribute Behaviour                          Implementation
+================= ================================== ==================================================================================
+
+``livetime``.        Sum of individual livetimes         :math:`\overline{t} = \sum_j t_j`
+
+``mask_safe``        Pixels added with `OR` operation     :math:`\overline{\epsilon_k} = \sum_{j} \epsilon_{jk}`
+
+``mask_fit``         Dropped
+
+``counts``           Summed outside exclusion region.     :math:`\overline{\mathrm{counts}_k} = \sum_j \mathrm{counts}_{jk} \cdot \epsilon_{jk}`
+
+``background``       Summed outside exclusion region.    :math:`\overline{\mathrm{bkg}_k} = \sum_j \mathrm{bkg}_{jk} \cdot \epsilon_{jk}`
+
+``exposure``         Summed outside spatial exclusion region    :math:`\overline{\mathrm{exposure}_l} = \sum_{j}{\mathrm{exposure}_{jl} \cdot \sum_k \epsilon_{jk}}`
+
+``psf``              Exposure weighted average                 :math:`\overline{\mathrm{psf}_l} = \frac{ \sum_{j}{\mathrm{psf}_{jl} \cdot \mathrm{exposure}_l} {\sum_{j} \cdot \mathrm{exposure}_l}`
+
+``edisp``            Exposure weighted average, with mask on reco energy :math:`\overline{\mathrm{edisp}_kl} = \frac{ \sum_{j}{\mathrm{edisp}_{jkl} \cdot \epsilon_{jk} \cdot \mathrm{exposure}_l} {\sum_{j} \cdot \mathrm{exposure}_l}`
+
+``gti``              Union of individual `gti`
+
+================= ================================== ==================================================================================
+
+
+
+It is important to keep in mind that:
+
+- Stacking happens in-place, ie, dataset1.stack(dataset2) will overwrite dataset
+- To properly handle masks, it is necessary to stack onto an empty dataset.
+- Stacking only works for maps with equivalent geometry.
+ Two geometries are called equivalent if one is exactly the same as,
+ or can be obtained from a cutout of, the other.
+- A stacked analysis is reasonable only when adding runs taken by the same instrument.
+
+
+Joint Analysis
```

Comment:
I think this definition is a better version of what we currently have in the glossary. So maybe just move to there for now and add a reference I the text above?